lib/scholar/options.ex

```diff
@@ -27,6 +27,22 @@ defmodule Scholar.Options do
     end
   end
 
+  def loss_function(function) do
+    case function do
+      function when is_function(function, 2) ->
+        {:ok, function}
+
+      nil ->
+        loss_fn = &Scholar.Linear.SVM.hinge_loss(&1, &2, c: 1.0, margin: 10)
```

Comment:
This shouldn't be in a general Scholar.Options module, because it's specific to SVM